packages/core-tabs/core-tabs.test.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import {mount} from 'enzyme'
+
+import Tabs from './core-tabs.jsx'
+
+const expectActiveTab = tab => {
+  expect(tab.getAttribute('aria-selected')).toEqual('true')
+}
+const expectInactiveTab = tab => {
+  expect(tab.getAttribute('aria-selected')).toEqual('false')
+}
+const expectActivePanel = panel => {
+  expect(panel.hasAttribute('hidden')).toBeFalsy()
+}
+const expectInactivePanel = panel => {
+  expect(panel.hasAttribute('hidden')).toBeTruthy()
+}
+
+describe('core-tabs/jsx', () => {
+  it('should have default props', () => {
```

Comment:
The `core-tabs.test.js` uses the phrase "**can** init on standard HTML" as opposed to "**should** have default props". This is a minor issue, however, I think we should have consistent phrasing. My personal preference is **should**.

Reply:
I agree, updated